democracy_club/templates/about/index.html

```diff
@@ -18,31 +18,24 @@
-since 2017, we have also worked with local and national government across all four
-UK nations.
+2015. We now run a national online polling station finder and candidate database, serving election information to [millions of UK](https://democracyclub.org.uk/research/impact/) voters earch year.
 
 In our short history we have transformed the accessibility of UK elections. We've
 opened up the online civic space, providing [non-partisan platforms](https://democracyclub.org.uk/blog/2023/02/17/democracy-club-and-political-impartiality/) which enable
 citizens to scrutinise, evaluate, and engage directly with their politicians and
 candidates. We've delivered personalised election information to well over ten
 million people since 2018. In a world which is increasingly concerned about the
 effect of the internet on democracy, we're using digital and data to inform and
-empower. [Read our current organisational strategy](https://democracyclub.org.uk/blog/2022/01/05/2022-and-beyond-an-updated-strategy/).
-
-Working alongside other organisations is at the core of what we do. Our tools are
-used widely across UK civil society, from student unions and civic education
-organisations, through to technology and social and traditional media companies.
-During 2019 we sent 218m notifications in partnership with Facebook, Instagram,
-Twitter, and SnapChat to remind people to vote. We've also enabled groups supporting
-the most marginalised to get their members engaged and voting, such as migrants,
-Roma and Travellers, and people with learning disabilities and autism. Furthermore,
-by making data on elections more freely available we've enabled high-quality
-research and journalism - for example, The Times and the Fawcett Society have
-recently used our data to shine a light on the 'glacial' progress of gender and
-age diversity among election candidates.
+empower. 
+
+**[Read our current organisational strategy](https://democracyclub.org.uk/blog/2022/01/05/2022-and-beyond-an-updated-strategy/).**
+
+Working alongside other organisations is at the core of what we do. A data supplier to the [Electoral
+Commission](https://democracyclub.org.uk/blog/2023/10/25/case-study-electoral-commission/)
+since 2017, we have also worked with local and national government across all four
+UK nations. 
+
+Working alongside other organisations is at the core of what we do. Our tools are used widely across UK civil society, from student unions and civic education organisations through to social media companies.  We've also enabled groups supporting the most marginalised to get their members engaged and voting, such as migrants, Roma and Travellers, and people with learning disabilities and autism. Our candidate data is [free to download](https://democracyclub.org.uk/data_apis/data/), and our postcode lookup can be accessed via [an API](https://democracyclub.org.uk/data_apis/voting_information_api/), allowing other organisations to build their own tools.
```

Comment:
We are repeating the sentence "Working alongside other organisations is at the core of what we do." twice here